views/show_folder.templ

```diff
@@ -33,9 +33,16 @@ templ ShowFolder(c *ctx.Ctx, folder *models.Folder) {
 						</div>
 						<div class="bc-toolbar-right">
 							<div class="bc-toolbar-item">
-								<a class="btn btn-link btn-link-muted" href={ templ.URL(c.Path("space", "spaceName", folder.Space.Name).String()) }>
+								<a
+									class="btn btn-link btn-link-muted"
+									href={ templ.URL(c.Path("space", "spaceName", folder.Space.Name, "focus", "new-folder").String()) }
```

Comment:
The last two arguments translate to a `?focus=new-folder` query string that is picked up here:
https://github.com/ugent-library/deliver/blob/feature/new-folder-hotkey/handlers/spaces.go#L168-170

This means this is visible in the URL.